h/events.py

```diff
@@ -9,6 +9,9 @@ def __init__(self, request, annotation_id, action):
         self.annotation_id = annotation_id
         self.action = action
 
+    def __eq__(self, other):
+        return self.__dict__ == other.__dict__
```

Comment:
We should add tests for this method to make sure that it actually does what it's supposed to.